### PR TITLE
[feat] linter eslint

### PR DESCRIPTION
**Pull request by Tristancalard**

### CODE REVIEW

**Review by thmarinho — COMMENTED** [review]
Is the linter setup on on the frontend as well ?

**Comment by Tristancalard** [comment]
> Is the linter setup on on the frontend as well ?

This is the linter for the backend

File: backend/src/entity/inventory.ts [hunk 1]
```diff
@@ -3,23 +3,23 @@ import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
...
+        quantity: number;
 
     @Column({ type: 'boolean', nullable: false, default: true })
-    available: boolean;
+        available: boolean;
 
     @Column({ type: 'text', nullable: false })
-    description: string;
+        description: string;
 }
```
Comment (alanschnegg) [inline backend/src/entity/inventory.ts]: Why is they're intentation here ?

**Review by alanschnegg — CHANGES_REQUESTED** [review]

File: backend/src/entity/inventory.ts (hunk 1 above)
Comment (Tristancalard) [reply backend/src/entity/inventory.ts]: linter wants it

Comment (alanschnegg) [reply backend/src/entity/inventory.ts]: We do not

Comment (Tristancalard) [reply backend/src/entity/inventory.ts]: it's done

**Review by alanschnegg — APPROVED** [review]

**Pull request merged**